source/includes/_this.md

```diff
@@ -0,0 +1,26 @@
+#This
+
+```javascript
+	
+	// 'this' in a global context refers
+	// to the global object, in this case, the document.
+
+	console.log(this.document === document); //true 
```

Comment:
I don't know if this is a particularly useful example. Where did document come from?